apps/polls/assets/QuestionForm.jsx

```diff
@@ -56,10 +56,9 @@ let QuestionForm = React.createClass({
                 htmlFor={'id_questions-' + this.props.key + '-name'}>
                 {django.gettext('Question:')}
               </label>
-              <input
+              <textarea
```

Comment:
No, the max length is more than generous. It was just changed to a textarea because of the design.